pkg/policy/policies_iterator.go

```diff
@@ -0,0 +1,54 @@
+package policy
+
+import (
+	"github.com/aquasecurity/tracee/pkg/utils"
+)
+
+// PoliciesIterator is an iterator for Policies.
+type PoliciesIterator struct {
+	policies []*Policy
+	index    int
+}
+
+// HasNext returns true if there are more policies to iterate.
+func (i *PoliciesIterator) HasNext() bool {
+	return i.index < len(i.policies)
+}
+
+// GetNext returns the next policy in the iteration.
+func (i *PoliciesIterator) GetNext() *Policy {
+	if i.HasNext() {
```

Comment:
stylistic nit to revert the conditional and short return nil here.

Reply:
Good point. Sticking to the happy path whenever possible.